### DISCUSSION TITLE
How do you open the GUI from the notification tray?

### DISCUSSION THREAD

**Discussion by cmacsithigh** [rancher-sandbox/rancher-desktop#2756] [Q&A]
Hey,

I'm loving the app as I've made the switch from Docker Desktop. However, I can't see to open the GUI from the notification tray by either clicking or double clicking on the icon. This functionality is pretty standard in most apps (Docker desktop, slack etc.)

The hacky way to open is it by clicking on preferences

**Comment by jandubois** [comment]
Hi @cian-sheehy, which OS are you using? At least on macOS it doesn't seem standard that clicking/double-clicking on notification icons opens the app; it mostly opens the context menu.

**Reply by cmacsithigh** [reply]
Hi @jandubois,

On windows, at least from my experience clicking/doubling clicking always opens up the app. I can share a gif of the experience with zoom and spotify

![tray](https://user-images.githubusercontent.com/30445066/187435778-3659cf31-ac1d-47e3-ba40-be4e4164268b.gif)
